gammapy/spectrum/powerlaw.py

```diff
@@ -34,7 +34,6 @@
     'power_law_pivot_energy',
     'power_law_df_over_f',
     'power_law_flux',
-    'power_law_energy_flux',
```

Comment:
Please don't remove this now. This breaks other parts in Gammapy and other scripts:
https://ci.appveyor.com/project/cdeil/gammapy/build/1.0.1122/job/4ki3h7p55hew82pn#L1157
I'll remove it later.